mypyc/buildc.py

```diff
@@ -103,6 +99,13 @@ def include_dir() -> str:
 # library in the directory that they live in.
 elif sys.platform == 'linux':
     vars['LDSHARED'] += ' -Wl,-rpath,"$ORIGIN"'
+    extra_compile_args += ['-Wno-unused-but-set-variable']
```

Comment:
Could you add a comment saying that this is janky and doesn't work with clang and we'll need to do something smarter?